module/Database/src/Controller/MemberController.php

```diff
@@ -258,9 +258,24 @@ public function showAction(): ViewModel
             return $this->memberIsDeleted($member);
         }
 
+        $noteForm = $this->memberService->getAuditNoteForm($member);
+        if ($this->getRequest()->isPost() && 'new-auditentry' === $this->getRequest()->getPost('submit', '')) {
+            $noteForm->setData($this->getRequest()->getPost()->toArray());
+
+            if ($noteForm->isValid()) {
+                $this->memberService->addAuditNote($member, $noteForm);
+                $this->flashMessenger()->addSuccessMessage('Note has been added to member');
+
+                return $this->redirect()->toRoute('member/show', [
+                    'id' => $lidnr,
+                ]);
+            }
+        }
+
```

Comment:
I think this should have been a separate action but it works as expected (the `redirect()` is necessary to ensure that the `flashMessenger` is successfully applied on the page).